herddb-core/src/main/java/herddb/core/TableSpaceManager.java

```diff
@@ -1602,6 +1624,35 @@ public TableSpaceCheckpoint(
             this.tablesCheckpoints = tablesCheckpoints;
         }
     }
+    
+    public void createAndWriteTableDigest(TableSpaceManager manager, String tableSpace,String table ){
+        CommitLogResult pos;
+        boolean lockAcquired = false;   
+        StatementEvaluationContext context =  StatementEvaluationContext.DEFAULT_EVALUATION_CONTEXT(); 
+        long id = newTransactionId.incrementAndGet();
```

Comment:
you don't need to start a transaction here

Reply:
Right, I move it into the try catch before create a digest